ajax/libs/pnp-common/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "pnp-common",
+  "filename": "common.es5.umd.js",
```

Comment:
I suppose this should better be `common.es5.umd.min.js`? Same as the other libraries.